indico/modules/events/models/persons.py

```diff
@@ -356,10 +356,22 @@ def person(cls):
             )
         )
 
+    @declared_attr
+    def display_order(cls):
+        return db.Column(
+            db.Integer,
+            nullable=False,
+            default=0
+        )
+
     @property
     def email(self):
         return self.person.email
 
+    @property
+    def display_order_key(self):
+        return (self.display_order, self.first_name, self.last_name)
```

Comment:
![](http://image.slidesharecdn.com/thezenofpython-140321035634-phpapp01/95/the-zen-of-python-3-638.jpg?cb=1395374577)